### PR TITLE
Expand suppression info for quality rules

### PR DESCRIPTION
**Pull request by gewarren**
Adds preprocessor directive and category-specific suppression examples specific to each code quality rule.

Contributes to https://github.com/dotnet/docs/issues/24437.

[Preview link](https://review.docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1002?branch=pr-en-us-29826#suppress-a-warning).

cc @Youssef1313

### CODE REVIEW

File: docs/fundamentals/code-analysis/quality-rules/ca5350.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ ms.author: gewarren
...
 
-| | Value |
-|-|-|
-| **Rule ID** |CA5350|
-| **Category** |[Microsoft.Cryptography](security-warnings.md)|
-| **Fix is breaking or non-breaking** |Non-breaking|
+|                                     | Value                                |
+|-------------------------------------|--------------------------------------|
+| **Rule ID**                         | CA5350                               |
+| **Category**                        | [Cryptography](security-warnings.md) |
```
Comment (Youssef1313) [inline docs/fundamentals/code-analysis/quality-rules/ca5350.md:14]:
```suggestion
| **Category**                        | [Security](security-warnings.md) |
```

File: docs/fundamentals/code-analysis/quality-rules/ca5350.md [hunk 2]
```diff
@@ -43,14 +43,29 @@ Suppress a warning from this rule when the level of protection needed for the da
...
 [*.{cs,vb}]
 dotnet_diagnostic.CA5350.severity = none
 ```
 
-For more information and to find other ways to suppress a rule, such as with a preprocessor directive, see [How to suppress code analysis warnings](../suppress-warnings.md).
+To disable this entire category of rules, set the severity for the category to `none` in the [configuration file](../configuration-files.md).
+
+```ini
+[*.{cs,vb}]
+dotnet_analyzer_diagnostic.category-Cryptography.severity = none
```
Comment (Youssef1313) [inline docs/fundamentals/code-analysis/quality-rules/ca5350.md:65]:
```suggestion
dotnet_analyzer_diagnostic.category-Security.severity = none
```

**Review by BillWagner — APPROVED** [review]
Thanks @gewarren 

This LGTM, once you resolve the conversation with @Youssef1313

**Comment by gewarren** [comment]
Thanks @BillWagner. Which conversation are you referring to?

**Comment by BillWagner** [comment]
> Thanks @BillWagner. Which conversation are you referring to?

It looks like I loaded the page before you accepted the suggestions. :shipit:

**Pull request merged**